[mygui] Build error on x64-osx

Package: mygui[core]:x64-osx -> 3.4.1#2
**Host Environment**
- Host: arm64-osx
- Compiler: AppleClang 13.0.0.13000029
-    vcpkg-tool version: 2999-12-31-unknownhash-debug
    vcpkg-scripts version: 4fff1e9cf 2022-01-17 (16 hours ago)


**To Reproduce**
`vcpkg install `


**Failure logs**
```
-- Using cached MyGUI-mygui-81e5c67e92920607d16bc2aee1ac32f6fd7d446b.tar.gz.
-- Extracting source /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/downloads/MyGUI-mygui-81e5c67e92920607d16bc2aee1ac32f6fd7d446b.tar.gz
-- Applying patch fix-generation.patch
-- Using source at /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean
-- Found external ninja('1.10.2').
-- Configuring x64-osx-dbg
-- Configuring x64-osx-rel
-- Building x64-osx-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:157 (message):
    Command failed: /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/downloads/tools/cmake-3.21.1-osx/cmake-3.21.1-macos-universal/CMake.app/Contents/bin/cmake --build . --config Debug --target install -- -v -j11
    Working Directory: /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/x64-osx-dbg
    See logs for more information:
      /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/install-x64-osx-dbg-out.log

Call Stack (most recent call first):
  /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/share/vcpkg-cmake/vcpkg_cmake_build.cmake:102 (vcpkg_execute_build_process)
  /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/share/vcpkg-cmake/vcpkg_cmake_install.cmake:41 (vcpkg_cmake_build)
  ports/mygui/portfile.cmake:35 (vcpkg_cmake_install)
  scripts/ports.cmake:145 (include)



```
<details><summary>/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/install-x64-osx-dbg-out.log</summary>

```
[1/107] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DMYGUI_STATIC -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/Platforms/Dummy/DummyPlatform/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/Common -fPIC -w -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -std=gnu++11 -MD -MT Platforms/Dummy/DummyPlatform/CMakeFiles/MyGUI.DummyPlatform.dir/src/MyGUI_DummyDataManager.cpp.o -MF Platforms/Dummy/DummyPlatform/CMakeFiles/MyGUI.DummyPlatform.dir/src/MyGUI_DummyDataManager.cpp.o.d -o Platforms/Dummy/DummyPlatform/CMakeFiles/MyGUI.DummyPlatform.dir/src/MyGUI_DummyDataManager.cpp.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/Platforms/Dummy/DummyPlatform/src/MyGUI_DummyDataManager.cpp
[2/107] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DMYGUI_STATIC -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/Plugins/Plugin_StrangeButton/. -fPIC -w -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk   -D_USRDLL -DMYGUI_BUILD_DLL -std=gnu++11 -MD -MT Plugins/Plugin_StrangeButton/CMakeFiles/Plugin_StrangeButton.dir/Plugin_Export.cpp.o -MF Plugins/Plugin_StrangeButton/CMakeFiles/Plugin_StrangeButton.dir/Plugin_Export.cpp.o.d -o Plugins/Plugin_StrangeButton/CMakeFiles/Plugin_StrangeButton.dir/Plugin_Export.cpp.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/Plugins/Plugin_StrangeButton/Plugin_Export.cpp
[3/107] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DMYGUI_BUILD -DMYGUI_STATIC -DMYGUI_USE_FREETYPE -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/include -I/usr/local/include -isystem /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -w -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -std=gnu++11 -Winvalid-pch -Xclang -emit-pch -Xclang -include -Xclang /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/x64-osx-dbg/MyGUIEngine/CMakeFiles/MyGUIEngine.dir/cmake_pch_x86_64.hxx -x c++-header -MD -MT MyGUIEngine/CMakeFiles/MyGUIEngine.dir/cmake_pch_x86_64.hxx.pch -MF MyGUIEngine/CMakeFiles/MyGUIEngine.dir/cmake_pch_x86_64.hxx.pch.d -o MyGUIEngine/CMakeFiles/MyGUIEngine.dir/cmake_pch_x86_64.hxx.pch -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/x64-osx-dbg/MyGUIEngine/CMakeFiles/MyGUIEngine.dir/cmake_pch_x86_64.hxx.cxx
...
Skipped 23 lines
...

[27/107] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DMYGUI_BUILD -DMYGUI_STATIC -DMYGUI_USE_FREETYPE -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/include -I/usr/local/include -isystem /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -w -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -std=gnu++11 -Winvalid-pch -Xarch_x86_64 -include/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/x64-osx-dbg/MyGUIEngine/CMakeFiles/MyGUIEngine.dir/cmake_pch_x86_64.hxx -MD -MT MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_FontManager.cpp.o -MF MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_FontManager.cpp.o.d -o MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_FontManager.cpp.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/src/MyGUI_FontManager.cpp
FAILED: MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_FontManager.cpp.o 
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DMYGUI_BUILD -DMYGUI_STATIC -DMYGUI_USE_FREETYPE -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/include -I/usr/local/include -isystem /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -w -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -std=gnu++11 -Winvalid-pch -Xarch_x86_64 -include/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/x64-osx-dbg/MyGUIEngine/CMakeFiles/MyGUIEngine.dir/cmake_pch_x86_64.hxx -MD -MT MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_FontManager.cpp.o -MF MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_FontManager.cpp.o.d -o MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_FontManager.cpp.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/src/MyGUI_FontManager.cpp
In file included from /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/src/MyGUI_FontManager.cpp:13:
In file included from /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/include/MyGUI_ResourceTrueTypeFont.h:15:
/usr/local/include/ft2build.h:37:10: fatal error: 'config/ftheader.h' file not found
#include "config/ftheader.h"
         ^~~~~~~~~~~~~~~~~~~
1 error generated.
[28/107] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DMYGUI_BUILD -DMYGUI_STATIC -DMYGUI_USE_FREETYPE -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/include -I/usr/local/include -isystem /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -w -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -std=gnu++11 -Winvalid-pch -Xarch_x86_64 -include/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/x64-osx-dbg/MyGUIEngine/CMakeFiles/MyGUIEngine.dir/cmake_pch_x86_64.hxx -MD -MT MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_Exception.cpp.o -MF MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_Exception.cpp.o.d -o MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_Exception.cpp.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/src/MyGUI_Exception.cpp
[29/107] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DMYGUI_BUILD -DMYGUI_STATIC -DMYGUI_USE_FREETYPE -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/include -I/usr/local/include -isystem /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -w -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -std=gnu++11 -Winvalid-pch -Xarch_x86_64 -include/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/x64-osx-dbg/MyGUIEngine/CMakeFiles/MyGUIEngine.dir/cmake_pch_x86_64.hxx -MD -MT MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_ControllerManager.cpp.o -MF MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_ControllerManager.cpp.o.d -o MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_ControllerManager.cpp.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/src/MyGUI_ControllerManager.cpp
[30/107] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DMYGUI_BUILD -DMYGUI_STATIC -DMYGUI_USE_FREETYPE -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/include -I/usr/local/include -isystem /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -w -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -std=gnu++11 -Winvalid-pch -Xarch_x86_64 -include/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/x64-osx-dbg/MyGUIEngine/CMakeFiles/MyGUIEngine.dir/cmake_pch_x86_64.hxx -MD -MT MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_FileLogListener.cpp.o -MF MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_FileLogListener.cpp.o.d -o MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_FileLogListener.cpp.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/src/MyGUI_FileLogListener.cpp
[31/107] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DMYGUI_BUILD -DMYGUI_STATIC -DMYGUI_USE_FREETYPE -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/include -I/usr/local/include -isystem /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -w -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -std=gnu++11 -Winvalid-pch -Xarch_x86_64 -include/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/x64-osx-dbg/MyGUIEngine/CMakeFiles/MyGUIEngine.dir/cmake_pch_x86_64.hxx -MD -MT MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_DDContainer.cpp.o -MF MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_DDContainer.cpp.o.d -o MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_DDContainer.cpp.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/src/MyGUI_DDContainer.cpp
[32/107] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DMYGUI_BUILD -DMYGUI_STATIC -DMYGUI_USE_FREETYPE -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/include -I/usr/local/include -isystem /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -w -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -std=gnu++11 -Winvalid-pch -Xarch_x86_64 -include/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/x64-osx-dbg/MyGUIEngine/CMakeFiles/MyGUIEngine.dir/cmake_pch_x86_64.hxx -MD -MT MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_FactoryManager.cpp.o -MF MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_FactoryManager.cpp.o.d -o MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_FactoryManager.cpp.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/src/MyGUI_FactoryManager.cpp
[33/107] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DMYGUI_BUILD -DMYGUI_STATIC -DMYGUI_USE_FREETYPE -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/include -I/usr/local/include -isystem /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -w -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -std=gnu++11 -Winvalid-pch -Xarch_x86_64 -include/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/x64-osx-dbg/MyGUIEngine/CMakeFiles/MyGUIEngine.dir/cmake_pch_x86_64.hxx -MD -MT MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_EditText.cpp.o -MF MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_EditText.cpp.o.d -o MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_EditText.cpp.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/src/MyGUI_EditText.cpp
[34/107] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DMYGUI_BUILD -DMYGUI_STATIC -DMYGUI_USE_FREETYPE -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/include -I/usr/local/include -isystem /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -w -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -std=gnu++11 -Winvalid-pch -Xarch_x86_64 -include/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/x64-osx-dbg/MyGUIEngine/CMakeFiles/MyGUIEngine.dir/cmake_pch_x86_64.hxx -MD -MT MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_DynLibManager.cpp.o -MF MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_DynLibManager.cpp.o.d -o MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_DynLibManager.cpp.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/src/MyGUI_DynLibManager.cpp
[35/107] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DMYGUI_BUILD -DMYGUI_STATIC -DMYGUI_USE_FREETYPE -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/include -I/usr/local/include -isystem /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -w -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -std=gnu++11 -Winvalid-pch -Xarch_x86_64 -include/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/x64-osx-dbg/MyGUIEngine/CMakeFiles/MyGUIEngine.dir/cmake_pch_x86_64.hxx -MD -MT MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_BackwardCompatibility.cpp.o -MF MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_BackwardCompatibility.cpp.o.d -o MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_BackwardCompatibility.cpp.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/src/MyGUI_BackwardCompatibility.cpp
[36/107] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DMYGUI_BUILD -DMYGUI_STATIC -DMYGUI_USE_FREETYPE -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/include -I/usr/local/include -isystem /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -w -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -std=gnu++11 -Winvalid-pch -Xarch_x86_64 -include/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/x64-osx-dbg/MyGUIEngine/CMakeFiles/MyGUIEngine.dir/cmake_pch_x86_64.hxx -MD -MT MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_ComboBox.cpp.o -MF MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_ComboBox.cpp.o.d -o MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_ComboBox.cpp.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/src/MyGUI_ComboBox.cpp
[37/107] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DMYGUI_BUILD -DMYGUI_STATIC -DMYGUI_USE_FREETYPE -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/include -I/usr/local/include -isystem /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -w -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -std=gnu++11 -Winvalid-pch -Xarch_x86_64 -include/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/x64-osx-dbg/MyGUIEngine/CMakeFiles/MyGUIEngine.dir/cmake_pch_x86_64.hxx -MD -MT MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_EditBox.cpp.o -MF MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_EditBox.cpp.o.d -o MyGUIEngine/CMakeFiles/MyGUIEngine.dir/src/MyGUI_EditBox.cpp.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mygui/src/f6fd7d446b-197296d096.clean/MyGUIEngine/src/MyGUI_EditBox.cpp
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**
<details><summary>vcpkg.json</summary>

```
{
  "name": "lichtsteuerung",
  "version-string": "2.0.0",
  "dependencies": [
    "mygui"
  ]
}

```
</details>

> /usr/local/include/ft2build.h:37:10: fatal error: 'config/ftheader.h' file not found

It picks freetype from brew. Probably because it adds `/usr/local` to the cmake prefix path.

--- a/ports/mygui/fix-osx-build.patch
+++ b/ports/mygui/fix-osx-build.patch
@@ -1,0 +1,25 @@
+From 0c1e0079653223fafb346f67c7e3bb001a39de73 Mon Sep 17 00:00:00 2001
+From: Leander Schulten <Leander.Schulten@rwth-aachen.de>
+Date: Sun, 23 Oct 2022 15:53:08 +0200
+Subject: [PATCH] do not simply include everything
+
+---
+ MyGUIEngine/CMakeLists.txt | 5 -----
+ 1 file changed, 5 deletions(-)
+
+diff --git a/MyGUIEngine/CMakeLists.txt b/MyGUIEngine/CMakeLists.txt
+index 33f74563b..40ec35101 100644
+--- a/MyGUIEngine/CMakeLists.txt
++++ b/MyGUIEngine/CMakeLists.txt
+@@ -48,11 +48,6 @@ if (MYGUI_USE_FREETYPE)
+ 	endif()
+ endif()
+ 
+-# platform specific dependencies
+-if (APPLE)
+-	include_directories(/usr/local/include /usr/include)
+-endif ()
+-
+ if (${CMAKE_VERSION} VERSION_EQUAL "3.16" OR ${CMAKE_VERSION} VERSION_GREATER "3.16")
+ 	target_precompile_headers(${PROJECTNAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/include/MyGUI_Precompiled.h")
+ endif ()

--- a/ports/mygui/portfile.cmake
+++ b/ports/mygui/portfile.cmake
@@ -12,6 +12,7 @@ vcpkg_from_github(
         fix-generation.patch
         Use-vcpkg-sdl2.patch
         Install-tools.patch
+        fix-osx-build.patch # from https://github.com/MyGUI/mygui/pull/244
 )
 
 if("opengl" IN_LIST FEATURES)

--- a/ports/mygui/vcpkg.json
+++ b/ports/mygui/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "mygui",
   "version": "3.4.1",
-  "port-version": 3,
+  "port-version": 4,
   "description": "Fast, flexible and simple GUI",
   "homepage": "http://mygui.info",
   "license": "MIT",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5030,7 +5030,7 @@
     },
     "mygui": {
       "baseline": "3.4.1",
-      "port-version": 3
+      "port-version": 4
     },
     "mysql-connector-cpp": {
       "baseline": "8.0.30",

--- a/versions/m-/mygui.json
+++ b/versions/m-/mygui.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "006d47cf87a29b68cbf5f7d1fbbc7e12a38e3cb7",
+      "version": "3.4.1",
+      "port-version": 4
+    },
+    {
       "git-tree": "5a7598c7df27293baca5e55e28c20aec14bf5672",
       "version": "3.4.1",
       "port-version": 3

